gadget-container/gadgettracermanager/main.go

```diff
@@ -36,6 +36,11 @@ import (
 	healthpb "google.golang.org/grpc/health/grpc_health_v1"
 	"google.golang.org/grpc/status"
 
+	// This is a blank include that actually imports all gadgets
+	// TODO: traceloop is imported separately because it is not in all-gadgets
+	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/all-gadgets"
+	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/traceloop/tracer"
```

Comment:
mmm, actually I don't think we need this here. Traceloop is currently only supported through the CR -> https://github.com/inspektor-gadget/inspektor-gadget/blob/255802f7d1e52b4d7cedbf497779f402c2fdbd69/pkg/all-gadgets/allgadgets.go#L17-L19. SO IIUC this import is not needed.